api/trace/always_sampler.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 const (
-	alwaysSamplerDescription = "AlwaysSampleSampler"
+	alwaysSamplerDescription = "AlwaysOnSampler"
```

Comment:
change the name of the variable as well. Perhaps `alwaysOnSamplerDescription`.
Same comment for AlwaysOffSampler.